### ISSUE TITLE
Days?

### ISSUE THREAD

**Issue by DarkSDragon** [szimek/final-countdown#1]
Is there any chance of adding a day countdown as well?

**Comment by szimek** [comment]
How long do you want to run this timer for? :) Of course it's possible, but it would add some complexity, as I'd need to hide days and hours in case there are none. 

I'm mainly using it to display time to speakers during talks at our meetups and all talks are shorter than an hour. Feel free to fork it and add it if if you need it.

**Comment by DarkSDragon** [comment]
Ahhh fair enough. Ill take a look at it and see what i can do.

Would be for project deadlines and such.

DD

**Comment by szimek** [comment]
It could be eventually added as an option - e.g. show hours, show days etc. and that would change how the mapping and rendering functions work. It would also need to affect the font size, to make sure that the whole thing can still fit in the screen.

**Comment by DarkSDragon** [comment]
True. If i get a spare moment ill have a look over and  see what i can come
up with. Im still getting familiar with git so working out the fork thing
lol
